src/util.js

```diff
@@ -54,3 +54,7 @@ exports.getPackageName = (branchName) => {
 
   return packageName
 }
+
+exports.isCommitHash = function(version) {
+  return /^[\w]{7}$/.test(version)
```

Comment:
this doesn't feel very robust. how do we know it will be always 7 chars? we don't!

Reply:
I've updated it to [this](https://github.com/fastify/github-action-merge-dependabot/pull/200/files#diff-9c0ec1b0a889e599f3ff81590864dd9dc65684a86b41f1da75acc53fafed12e3R70). Do you think that's alright?